scripts/git.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+git clone --depth=1 git@github.com:sass/libsass.git ./src/libsass
+cd ./src/libsass
+git checkout $LIBSASS_GIT_VERSION
```

Comment:
This is required be cause we have no way to know which commit the submodule is linked to when npm installs via git https://github.com/npm/npm/issues/6700#issuecomment-71302066